app/badge/serializers/leaderboard.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import serializers
+
+from app.badge.models import UserBadge
+from app.common.serializers import BaseModelSerializer
+from app.content.models import User
+from app.content.serializers.user import DefaultUserSerializer
+
+
+class LeaderboardSerializer(BaseModelSerializer):
+    number_of_badges = serializers.SerializerMethodField()
+    user = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = (
+            "user",
+            "number_of_badges",
+        )
+
+    def get_user(self, obj):
+        return DefaultUserSerializer(obj).data
+
+    def get_number_of_badges(self, obj):
+        return obj.user_badges.count()
```

Comment:
Når denne brukes for ledertavle i badge-kategori så tar den ikke hensyn (såvidt jeg klarer å se) til at en kun ønsker antall badges hver bruker har innenfor kategorien. Per nå ser det ut som vi kun viser brukere som faktisk har en badge i kategorien (riktig), men så viser antall badges hver bruker har totalt blant absolutt alle badges.

Tar jeg feil?